fontforgeexe/startui.c

```diff
@@ -1080,12 +1080,11 @@ int fontforge_main( int argc, char **argv ) {
     textdomain("FontForge");
     GResourceUseGetText();
     {
-	char path[PATH_MAX];
-	snprintf(path, PATH_MAX, "%s%s", getShareDir(), "/pixmaps" );
-	GGadgetSetImageDir( path );
+	GGadgetSetImageDir( getPixmapDir() );
```

Comment:
GGadgetSetImageDir leaves its argument untouched. I think that we need to free it here.